runzero-api.yml

```diff
@@ -12,13 +12,17 @@ info:
     name: Commercial
     url: 'https://www.runzero.com/legal/terms'
 tags:
+  - name: Account
+    description: Requires Account Key starting with CT or OAuth
   - name: Organization
-    description: Requires Organization Key
+    description: Requires Organization Key starting with OT, Account Key, or OAuth
   - name: Export
-    description: Requires Export Token or Organization Key
+    description: Requires Export Token starting with ET, Account Key, or Organization Key
+  - name: Import
+    description: Requires Account Key, Organization Key, or OAuth
```

Comment:
Let's make these a bit more consistent.

- "Requires Account key (starts with CT), or OAuth"
- "Requires Organization key (starts with OT), Account key (starts with CT), or OAuth"
- "Requires Export token (starts with ET), Organization key (starts with OT), Account key (starts with CT), or OAuth"

Oxford comma, list the keys that are smaller scope first, provide clues for recognizing key types in all cases.